eras/alonzo/impl/testlib/Test/Cardano/Ledger/Alonzo/Binary/TxWitsSpec.hs

```diff
@@ -88,6 +88,7 @@ plutusScriptsProp = do
     keys SPlutusV1 = 3
     keys SPlutusV2 = 6
     keys SPlutusV3 = 7
+    keys SPlutusV4 = 7 -- TODO: investigate
```

Comment:
What will be the number of keys for `V4`, do we know?

Reply:
This is a tag in witnesses, so it will be `8`. 
See other tags here, which is also the place that needs to be adjusted for PlutusV4
https://github.com/IntersectMBO/cardano-ledger/blob/ecd0f677392a37d1e45e91590e117d4c3db819db/eras/alonzo/impl/src/Cardano/Ledger/Alonzo/TxWits.hs#L558-L562
```suggestion
    keys SPlutusV4 = 8
```